pkg/inventory/inventory_test.go

```diff
@@ -414,10 +413,3 @@ func TestLegacyInventoryName(t *testing.T) {
 func copyInventoryInfo() *unstructured.Unstructured {
```

Comment:
Named Info and returning Unstructured is weird.

Perhaps just remove this method and use `inventoryObj.DeepCopy()` as needed.